kpc/forms.py

```diff
@@ -46,7 +38,7 @@ def __init__(self, *args, **kwargs):
                 self.fields['contact'].initial = contact
         else:
             self.fields['contact'].choices = [('', 'Select a licensee')]
-        self.fields['cert_from'].initial = get_latest_cert()
+        self.fields['cert_from'].initial = Certificate.objects.next_available()
```

Comment:
I generally like custom querysets and managers, but this one seems a little funky. Perhaps it's just wording, but I'd expect `Certificate.objects.next_available()` to return a `Certificate`. I wonder if a class method on `Certificate` would telegraph the intent a bit better?

Reply:
Good point, implementation is much more concise as well.